.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: python
 python:
-  - "2.7"
+  - "3.8"
 install:
-  - pip install pycodestyle==2.3.1 flake8
+  - pip install pycodestyle==2.6.0 flake8
   - pip check
+  - CASS_DRIVER_NO_CYTHON=1 pip install -r requirements.txt
 script:
   # we want pyflakes to check all files for unused imports only
   # we use flake8 because it allows us to ignore other warnings
   # exclude the thrift directories - they contain auto-generated code
-  - flake8 --ignore=E501,F811,F812,F822,F823,F831,F841,N8,C9 --exclude=thrift_bindings,cassandra-thrift .
+  # - flake8 --ignore=E501,F811,F812,F822,F823,F831,F841,N8,C9 --exclude=thrift_bindings,cassandra-thrift .
```

Comment:
instead of commenting it out, could we add more ignores (for those currently failing?)

Reply:
I don't know if you tried, but there are so many of them that is probably make no sense at all - instead, I can spend 1-2 hours on fixing that properly

Reply:
> I can spend 1-2 hours on fixing that properly

i have a PR somewhere for that :-)
(it would need to be a separate ticket.)

let's just comment out the line then for now 👍